lib/tools/apk-utils.js

```diff
@@ -241,21 +241,25 @@ apkUtilsMethods.waitForNotActivity = async function (pkg, act, waitMs = 20000) {
  */
 apkUtilsMethods.uninstallApk = async function (pkg) {
   log.debug(`Uninstalling ${pkg}`);
+  if (!this.isAppInstalled(pkg)) {
+    log.info(`${pkg} was not uninstalled, because it was not present on the device`);
+    return false;
```

Comment:
Is this `false` taken as an error condition anywhere?

Reply:
does this matter? These changes don't affect the previous logic of this mathod

Reply:
Does it not? If the app is not installed the uninstall process doesn't get attempted. That seems like a change in the logic of this method.

I'm just wanting to make sure that when `false` is returned the calling code does not consider it to have been a problem.

Reply:
The result is only used in the removeApp API call inside general.js:

```
$ cd appium-android-driver/lib/
$ grep -R 'uninstallApk' *
android-helpers.js:      await adb.uninstallApk(pkg);
commands/general.js:  return this.adb.uninstallApk(appPackage);
commands/general.js:    await this.adb.uninstallApk(this.opts.appPackage);
driver.js:      await this.adb.uninstallApk(this.opts.appPackage);
driver.js:        await this.adb.uninstallApk(this.opts.appPackage);
$ cd appium-uiautomator2-driver/lib/
$ grep -R 'uninstallApk' *
driver.js:      await this.adb.uninstallApk(this.opts.appPackage);
driver.js:          await this.adb.uninstallApk(this.opts.appPackage);
uiautomator2.js:      await this.adb.uninstallApk(apkPackage);
uiautomator2.js:      await this.adb.uninstallApk(testApkPackage);
```

Reply:
```
$ cd appium-selendroid-driver/lib/
$ grep -R 'uninstallApk' *
driver.js:      await this.adb.uninstallApk(this.opts.appPackage);
selendroid.js:      await this.adb.uninstallApk(this.modServerPkg);
```

Reply:
Excellent. Thanks!